site/content/en/docs/tasks/troubleshooting/troubleshooting_delete_clusterqueue.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Troubleshooting delete ClusterQueue"
+date: 2024-08-28
+weight: 5
+---
+
+Deleting a ClusterQueue can be a bit complex when there are active Workloads. 
+
+This guide provides a clear approach to safely deleting a ClusterQueue,
+explaining the process and offering alternative methods to handle specific scenarios.
+
+For this example, let's assume your ClusterQueue is named `my-cq`.
+
+## Understanding `kubectl delete clusterqueue`
+
+When you run:
+
+```sh
+kubectl delete clusterqueue my-cq
+```
+
+you are initiating the deletion of the ClusterQueue object. 
+However, if there are Workloads still utilizing the ClusterQueue, the command may hang or take some time to complete.
+
+
+## Why does this happen?
+
+Kueue creates a [Workload](/docs/concepts/workload/) object for each Job to track its admission status. 
+If a Workload is admitted, it is associated with a specific ClusterQueue. 
+Kueue uses a [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) 
+called kueue.x-k8s.io/resource-in-use to prevent deletion of the ClusterQueue while resources are still in use. 
+Consequently, a ClusterQueue with this finalizer cannot be deleted until all resources are released.
+
+
+## How to resolve this issue?
```

Comment:
Maybe we can remove this section and just leave `How to stop the ClusterQueue?` below.